### PR TITLE
Update dependencies, include asset file in PyPI

### LINKED ISSUE #72: Missing dependency: gevent (fix known)
`gevent` is [used directly](https://github.com/pipermerriam/ethereum-alarm-clock/search?utf8=%E2%9C%93&q=gevent&type=) by EAC, but not listed as such in `setup.py`.

Due to that, startup fails if following [the docs](http://docs.ethereum-alarm-clock.com/en/latest/cli.html#).

Fix should be a one-liner, won't submit now due to #73.

### PR DESCRIPTION
**Pull request by veox**
### Summary

* Fixes #72, #73, #74.
* Lists `gevent` as a hard dependency.
* Updates to newer Populus.
* Includes a file missing from PyPI distribution.
* Details in commit messages, or (very verbose) in linked issues.

### Notes

* **CI is likely to fail before getting to check these changes** - will see what I can do. Probably safer to fix that first (see issue #77).
* There seems to have been a typo when preparing a PyPI package of `ethereum-alarm-clock-client`: a version of `v8.0.0b1` was tagged, where `v0.8.0b1` was more likely. If this was indeed a mistake, **the release should be yanked**, which may require fiddling with both Github and PyPI.

### Animal picture

A slightly concerned walrus ([via reddit](https://www.reddit.com/r/photoshopbattles/comments/3ai69v/psbattle_walrus_cub/)):

Young walrus with huge moustache sticks her head out of the water

### CODE REVIEW

**Comment by veox** [comment]
To clarify, I haven't been able to start the client with these changes (see [this comment](https://github.com/pipermerriam/ethereum-alarm-clock/issues/74#issuecomment-294967454) in issue #74), but they do seem to be moving in the right direction.

My node's on mainnet, and since EAC v0.8.0 hasn't been deployed there, I've got no way to check (yet) if it breaks hard. Might set up another one on testnet to check, if time permits.

File: MANIFEST.in [hunk 1]
```diff
@@ -6,4 +6,4 @@ include requirements.txt
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
 
-include alarm_client/assets/v8.0.0.json
+include alarm_client/assets/v0.8.0.json
```
Comment (pipermerriam) [inline MANIFEST.in]: can you change this to be `include alarm_client/assets/*.json` so that this file doesn't need to be updated for each release?

Comment (veox) [inline MANIFEST.in]: Sure, one sec...

Comment (veox) [inline MANIFEST.in]: Done. :)

**Comment by veox** [comment]
Made requested change, plus updated dependency versions in `requirements.txt`, too - not just `setup.py`.

**Pull request merged**